### ISSUE TITLE
Transaction Fee Calculation is pretty expensive

### ISSUE THREAD

**Issue by AdamBLevine** [loon3/LTB-Companion#6]
If my math is correct at .00043230 per transaction thats like 10 cents each.  

Devon and I talked about how much it costs to send within our system in a [swapbot issue](https://github.com/tokenly/swapbot/issues/10)

>Using these tiny fees (which can be changed):
>BTC Dust: 0.0000543
>Fee: 0.0001
>Escrow Dust: 0.000025

which comes to three cents per tx and is more reasonable.

**Comment by loon3** [comment]
I haven't started exploring the malleability of XCP asset transactions fees (which I'm sure you guys are much more familiar with).  The 0.00043230 BTC number is what it cost me to perform an asset send in Counterwallet when I was putting this feature together.  

The cheaper the better; I'll work with Devon on transaction construction once I'm at that point.

**Comment by deweller** [comment]
Yeah - we can definitely use fees below the Counterwallet defaults.  But the tradeoff is that transactions with low fees can take a while (especially if the transaction will use a bunch of small UTXOs).

We will likely need a "high priority" transaction option at some point, but not for the initial prototype.

**Comment by AdamBLevine** [comment]
Great idea on the "High Priority" checkable option that increases the miner fee for quicker inclusion.
